packages/core/src/command/ai-brief/ai-brief-shared.ts

```diff
@@ -0,0 +1,38 @@
+import { Binary, Bundle } from "@medplum/fhirtypes";
+import { Buffer } from "buffer";
+import dayjs from "dayjs";
+import { uuidv7 } from "../../util/uuid-v7";
+
+export async function generateAiBriefFhirResource(
+  content: string | undefined
+): Promise<Binary | undefined> {
+  if (!content) return undefined;
+
+  const encodedContent = Buffer.from(content).toString("base64");
+
+  return {
+    resourceType: "Binary",
+    id: uuidv7(),
+    meta: {
+      versionId: "1",
+      lastUpdated: dayjs().toISOString(),
+      source: "metriport:ai-generated-brief",
+    },
+    contentType: "text/plain",
+    data: encodedContent,
+  };
+}
+
+export function getAiBriefContentFromBundle(bundle: Bundle): string | undefined {
+  const binaryResourceEntry = bundle.entry?.find(
+    entry => entry.resource?.resourceType === "Binary"
```

Comment:
Could create a `isBinary` like we have `isPractitioner` and others, then we also don't need the casting below